frontend/components/DocumentFilters/DocumentFilters.jsx

```diff
@@ -140,9 +166,8 @@ export default class DocumentFilters extends Component {
     const {updateUrlParams} = this.props
 
     let validFilters = filters.filter(entry => {
-      let isValid = Object.keys(entry).filter(key => {
-        return Object.is(entry[key], null)
-      })
+      let isValid = Object.keys(entry)
+        .filter(key => Object.is(entry[key], null))
```

Comment:
Shouldn't that decision be made when the cost of supporting IE has an impact on efficiency/maintainability? I see no reason to use `Object.is` in this case or in the case I linked to, which should be a simple comparison.